research/src/pages/breadcrumb.proposal.mdx

```diff
@@ -0,0 +1,60 @@
+---
+menu: Components
+name: Breadcrumb (Editor's Draft)
+path: /components/breadcrumb
+pathToResearch: /components/breadcrumb.research
+---
+
+import BreadcrumbAnatomy from '../components/breadcrumb-anatomy'
+
+## Overview
+
+Breadcrumbs are navigation land marks used to show a user's location within the site or app.
+
+## Use Cases
+
+A `<breadcrumb />` trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place within a website or web application. Breadcrumbs are often placed horizontally before a page's main content.
+
+## Prior Art/Examples <a href="#prior-art" id="prior-art"></a>
+
+- [Ant Design](https://ant.design/components/breadcrumb/)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/breadcrumb/)
+- [Auth0](https://styleguide.auth0.com/components/breadcrumb)
+- [Atlas Kit](https://atlassian.design/components/breadcrumbs/examples)
+- [Carbon](https://www.carbondesignsystem.com/components/breadcrumb/code/)
+- [Fabric](https://developer.microsoft.com/en-us/fluentui#/controls/web/breadcrumb)
+- [Lightning](https://www.lightningdesignsystem.com/components/breadcrumbs/)
+- [Semantic UI](https://semantic-ui.com/collections/breadcrumb.html)
+
+### Anatomy <a class="link" href="#anatomy" id="anatomy"></a>
+
+#### Diagram <a class="link" href="#diagram" id="diagram"></a>
+
+<BreadcrumbAnatomy />
+
+#### Structure <a class="link" href="#structure" id="structure"></a>
+
+- `<breadcrumb>` - The root element that contains the list of items and dividers, it's composed by `nav` and `ol` elements  **[required]**
+- `<breadcrumb-item>` - The element representing a place in the path to the current position on the application, it's a focusable `li` that can contain text or anchors **[required]**
+- `<breadcrumb-divider>` - The divider between the `<breadcrumb-item />`, it should be a non-focusable li
+
+Proposed HTML: 
```

Comment:
I don't think this is feasible, as effectively we're saying that a breadcrumb item is the `<li>` correct and the breadcrumb is the `nav`? I think it would be great if you and I can jump on a call to discuss this. DM me on twitter, I'm happy to setup a relatively early meeting to make this happen.

Reply:
I'd also expect that the anchor is the focusable portion here rather than the list item - I don't see why we'd have a non-focusable element receive focus as opposed to the anchor within listed below. Similarly, I'm not sure we'd want the divider/separator enumerated as a list item itself.

@gregwhitworth we're implementing Breadcrumb currently in FAST, it would be great to get visibility on this PR and some of the potential issues for the broader Open UI community.

Reply:
Hey @gregwhitworth 

This is the pattern proposed by aria 

```html
<nav aria-label="Breadcrumb" class="breadcrumb">
  <ol>
    <li>
      <a href="../../">
        WAI-ARIA Authoring Practices 1.1
      </a>
    </li>
....
```

Reply:
I'd push back on this a bit - this is an example by ARIA, but it's not "the way" IMO.

Reply:
`The following example demonstrates the breadcrumb design pattern.` Looks pretty heavy on the definition way for me. Also it's pretty much what was implemented by the libraries that has minimum a11y support for breadcrumbs.

Reply:
I don't want to get too far down the semantic rabbit hole, but I think that semantics are important - especially when it comes to the spec. The *spec* is the definition. The example, as you've noted above - _demostrates_ the design pattern. The spec itself points to that as an example. It is *one implementation* and I think we should be careful to conflate the example with the specification.

The spec on this is fairly simply as of now and so I'll paste below:
> - Breadcrumb trail is contained within a navigation landmark region.
> - The landmark region is labelled via aria-label or aria-labelledby.
> - The link to the current page has aria-current set to page. If the element representing the current page is not a link, aria-current is optional.

From an accessibility standpoint, the above is the minimum bar and the example is, as you mentioned, a very common implementation of the pattern following the spec.

Given the above, the important bit here is the role and hierarchy. While I agree we should aim to use native elements wherever possible, I believe that is too prescriptive an approach for us to take.

Reply:
I don't think there's anything particularly wrong with the DOM structure, but I would add that using the roles model is equally as valid here. There may be external influences, pre-existing implementations for teams, etc where using that exact structure may be prohibitive. I'm all good with focusing on the roles (nav, list, listitem) as the recommendation with an example of that using what you proposed above. That's really all I'm advocating for. The overall ARIA hierarchy vs. the implementation of that. Hopefully that clarifies what I mean.

Reply:
That do makes sense for me, but I think that what you see in the proposal or 

```html
<div role="navigation">
    <div role="list>
        <div role="listitem"></div>
    </div>
</div>
```

are equivalent and it's correct replacement. It's defined as "Proposed HTML" such as aria but not "The only correct way". Using the elements as in the example is the most generic way to represent the intention so it can be achieve as the same for other developers implementing it. In Fluent for example we are doing it only using divs and roles... But as a generic way I guess having the correct elements gives better the message.

Reply:
Yep, same page. :thumbsup:

Reply:
> @gregwhitworth we're implementing Breadcrumb currently in FAST, it would be great to get visibility on this PR and some of the potential issues for the broader Open UI community.

Feel free to ping on discord and raise the issues as separate issues (that you believe you need resolution on) to discuss on the call by adding them to the agenda by `agenda+` label